src/laravel/tests/Unit/Posts/CreatePostTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace App\tests\Unit\Posts;
+
+use App\Models\Image;
+use App\Models\Post;
+use App\Models\Tag;
+use App\Repositries\Posts\ImageRepository;
+use App\Repositries\Posts\PostRepository;
+use App\Repositries\Posts\PostTagRepository;
+use Database\Factories\PostFactory;
+use Database\Factories\UserFactory;
+use Database\Seeders\TagSeeder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\DB;
+use Tests\TestCase;
+
+class CreatePostTest extends TestCase
+{
+    use RefreshDatabase;
+    public function test_PostRepository_create__期待したPostオブジェクトが渡された時、postsテーブルにレコードを作成できていること()
+    {
+        $user = UserFactory::new()->create();
+        $testPost = Post::buildValidatedPostEntity(
+            title: 'test',
+            content: 'testです',
+            thumnail_image_url: '/public/post_thumnail_image/test.png',
+            user_id:  $user->id,
+        );
+
+        (new PostRepository)->create($testPost);
```

Comment:
`()`をつけて、明示的にコンストラクタを呼び出しましょう。

```suggestion
        (new PostRepository())->create($testPost);
```

---
あと、個人的には
```
$repo = new PostRepository();

$repo->create($testPost);
```
と分けるほうが好きです。`create()`をテストしたいわけですが、`new PostRepository()`をテストしたいわけではないので